src/assets/TestFramework/ArmDeployment.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+using System;
+
+namespace Azure.ResourceManager.TestFramework
+{
+    /// <summary> A Class representing a ArmDeployment along with the instance operations that can be performed on it. </summary>
+    public partial class ArmDeployment : ArmResource
+    {
+        /// <summary> Initializes a new instance of the <see cref="ArmDeployment"/> class for mocking. </summary>
+        public ArmDeployment()
+        {
+        }
+
+        public ArmDeploymentData Data;
+    }
+
+    public class ArmDeploymentData
+    {
+        public Property Properties;
+    }
+
+    public class Property
```

Comment:
In the real `Resources` RP, this should be `ArmDeploymentProperties` I believe? Could you please double check and change the name here accordingly?